app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] Let's remove commented code from your project. This makes it hard to read it.

Reply:
Implemented thank you for reviewing. :+1: